pkg/model/application.go

```diff
@@ -78,6 +78,17 @@ func (a *Application) ContainLabels(labels map[string]string) bool {
 	return true
 }
 
+func (a *Application) GetKindString() string {
```

Comment:
I wanted to use `GetKind`. But there is the same name method generated by protobuf.